web/src/site-admin/routes.tsx

```diff
@@ -51,6 +51,14 @@ export const siteAdminAreaRoutes: readonly SiteAdminAreaRoute[] = [
         render: lazyComponent(() => import('./SiteAdminRepositoriesPage'), 'SiteAdminRepositoriesPage'),
         exact: true,
     },
+    {
+        path: '/repositories-permissions',
```

Comment:
```suggestion
        path: '/repositories/permissions',
```

looks a bit nicer, WDYT?